verifiers/internal/gha/provenance.go

```diff
@@ -330,6 +330,13 @@ func isValidDelegatorBuilderID(prov iface.Provenance) error {
 		}
 	}
 
+	// Exception for slsa-framework/slsa-github-generator branches during testing mode
```

Comment:
I think I would prefer maybe your other suggestion that the user defines the verification repo path in their test rather than modifying the normal behavior of the code conditionally.